package.json

```diff
@@ -38,6 +38,6 @@
     "mocha": "^2.2.4",
     "node-mocks-http": "^1.2.3",
     "sinon": "^1.14.1",
-    "typescript": "^1.0.1"
+    "typescript": "^1.8.9"
```

Comment:
Where is the 1.8.9 version number recorded (I can only find documentation mentioning 1.8 and no revision number)?